src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,7 +1,59 @@
-import React from 'react';
+import React, {
+  useState,
+} from 'react';
 import './FindMovie.scss';
+import classNames from 'classnames';
+import {
+  getMovie,
+} from '../../api';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
+
+type Props = {
+  onAdd: (movie: Movie) => void,
+};
+
+export const FindMovie: React.FC<Props> = ({ onAdd }) => {
+  const [query, setQuery] = useState('');
+  const [newMovie, setNewMovie] = useState({} as Movie);
```

Comment:
```suggestion
  const [newMovie, setNewMovie] = useState<Movie>({
            title: '',
            description: '',
            imgUrl: '',
            imdbUrl: '',
            imdbId: '',
  });
```

better to set the default object in defaultState value

Reply:
you can move this object to some var and reuse it when you want to reset the state

Reply:
fixed)

Reply:
sorry, rolled it back, it brokes test passing